insights/parsers/sockstat.py

```diff
@@ -0,0 +1,117 @@
+"""
+SockStats - file ``/proc/net/sockstat``
+=======================================
+The ``TcpIpStats`` class implements the parsing of ``/proc/net/snmp``
+file, which contains TCP/IP stats of individual layer.
+"""
+
+
+from .. import Parser, parser, LegacyItemAccess
```

Comment:
Please use a uniform format for the `importing`s in one python module. E.g. all in relative path or all in full path

Reply:
Done